src/main/kotlin/org/partiql/scribe/sql/RexConverter.kt

```diff
@@ -414,64 +403,8 @@ public open class RexConverter(
         ctx: Unit,
     ): Expr {
         val inputRel = rex.input
-        val constructor = rex.constructor
-        return when (inputRel) {
-            is RelExcept -> relSetOpToBagOp(inputRel.left, inputRel.right, inputRel.isAll, SetOpType.EXCEPT(), ctx)
-            is RelIntersect -> relSetOpToBagOp(inputRel.left, inputRel.right, inputRel.isAll, SetOpType.INTERSECT(), ctx)
-            is RelUnion -> relSetOpToBagOp(inputRel.left, inputRel.right, inputRel.isAll, SetOpType.UNION(), ctx)
-            is RelProject -> {
-                val relConverter = transform.getRelConverter()
-                val rexConverter = transform.getRexConverter(locals)
-                val sfw = relConverter.apply(inputRel, ctx)
-                val rewrittenSelect = convertSelectValueToSqlSelect(sfw.select!!, inputRel)
-                sfw.select = rewrittenSelect
-                return exprQuerySet(
-                    body = sfw.toQueryBodySFW(),
-                )
-            }
-            else ->
-                listener.reportAndThrow(
-                    ScribeProblem.simpleError(
-                        code = ScribeProblem.UNSUPPORTED_PLAN_TO_AST_CONVERSION,
-                        message = "$inputRel is not yet supported",
-                    ),
-                )
-        }
-    }
-
-    private fun convertSelectValueToSqlSelect(
```

Comment:
self-review: this helper to convert `SELECT VALUE` into `SELECT` list was moved to `RelConverter` since it deals with rels more than rexs